<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdaterWindow</class>
 <widget class="QWidget" name="UpdaterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Crow Translate</string>
  </property>
  <property name="windowIcon">
   <iconset theme="software-update-available"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="changelogTextEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="progressLayout">
     <item>
      <widget class="QProgressBar" name="downloadBar"/>
     </item>
     <item>
      <widget class="QPushButton" name="cancelDownloadButton">
       <property name="toolTip">
        <string>Cancel download</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <widget class="QPushButton" name="downloadButton">
       <property name="toolTip">
        <string>Download the installer to the Downloads folder</string>
       </property>
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="installButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Exit the program and run the installer</string>
       </property>
       <property name="text">
        <string>Install</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="updateStatusLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateLaterButton">
       <property name="toolTip">
        <string>Close this window</string>
       </property>
       <property name="text">
        <string>Update later</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
